### PR TITLE
feat: add some docs, works OOTB

### PR DESCRIPTION
**Pull request by jaredallard**
**What this PR does**: Updates the docs and tests it E2E.

Also adds the ability to not set `zone_id` and use `auto_discovery` by default.

### CODE REVIEW

**Comment by jaredallard** [comment]
Closes #1, #2, and #3

File: roon-discord-publish.js [hunk 1]
```diff
@@ -179,21 +158,42 @@ const roon = new RoonApi({
...
+                activeZone = _transport._zones[settings.zone_id];
+            }
+
+            // We have no zone, set it to the latest "playing"
+            // zone.
+            if (activeZone === null) {
+                for (const zoneID of Object.keys(_transport._zones)) {
+                    const zone = _transport._zones[zoneID]
+                    if (zone.state === 'playing') {
+                        activeZone = zone
```
Comment (williamtdr) [inline roon-discord-publish.js:173]: nit: (and for the rest of the lines): semicolons

**Review by williamtdr — APPROVED** [review]

File: roon-discord-publish.js (hunk 1 above)
Comment (jaredallard) [reply roon-discord-publish.js:173]: I DON'T BELIEVE IN SEMICOLONS

**Comment by jaredallard** [comment]
@williamtdr pls merge AS IS

**Pull request closed without merging**